src/shader.cpp

```diff
@@ -220,47 +224,40 @@ class MainShaderConstantSetter : public IShaderConstantSetter
 		video::IVideoDriver *driver = services->getVideoDriver();
 		sanity_check(driver);
 
-		// set inverted world matrix
```

Comment:
Yes, sofar removed these and I didn't re-add them.  They don't seem to actually have any effect.